Cannot redeclare exported variable 'ZodSchema'

I seems that since typescript 4.9.x i get the following

```
node_modules/@asteasolutions/zod-to-openapi/dist/zod-extensions.d.ts:27:20 - error TS2323: Cannot redeclare exported variable 'ZodSchema'.

27     abstract class ZodSchema<Output, Def extends ZodTypeDef, Input = Output> {
```

--- a/src/zod-extensions.ts
+++ b/src/zod-extensions.ts
@@ -27,11 +27,7 @@ declare module 'zod' {
     openapi?: ZodOpenApiFullMetadata;
   }
 
-  export abstract class ZodSchema<
-    Output,
-    Def extends ZodTypeDef,
-    Input = Output
-  > {
+  interface ZodSchema<Output, Def extends ZodTypeDef, Input = Output> {
     openapi<T extends ZodSchema<any>>(
       this: T,
       metadata: Partial<ZodOpenAPIMetadata<z.infer<T>>>

--- a/src/zod-extensions.ts
+++ b/src/zod-extensions.ts
@@ -27,7 +27,11 @@ declare module 'zod' {
     openapi?: ZodOpenApiFullMetadata;
   }
 
-  abstract class ZodSchema<Output, Def extends ZodTypeDef, Input = Output> {
+  export abstract class ZodSchema<
+    Output,
+    Def extends ZodTypeDef,
+    Input = Output
+  > {
     openapi<T extends ZodSchema<any>>(
       this: T,
       metadata: Partial<ZodOpenAPIMetadata<z.infer<T>>>

